packages/core/src/view/handler/ConstraintHandler.ts

```diff
@@ -38,7 +40,7 @@ class ConstraintHandler {
    * {@link Image} to be used as the image for fixed connection points.
    */
   // pointImage: mxImage;
-  pointImage = new Image(`${Client.imageBasePath}/point.gif`, 5, 5);
+  pointImage = new Image(pointImg, 5, 5);
```

Comment:
⚠️ this prevents to use an alternate image
Historically, the images were not packaged withing the mxGraph bundle. When using the editor, you could provide your own image set and `mxClient.imageBasePath` was used to configure the path to the custom images. The same remark applies to the change in `ConnectionHandler`.
My comment is mainly about inform all of us that we drop this support for sure. I don't know if it was broken in maxGraph before this change.

Reply:
I think it'd better to provide a way to overwrite those images by adding some parameters to Graph.
Maybe we could change the signature of Graph constructor a bit like (to an object),
```js
constructor({
  container: ...,
  model: ...,
  plugins: ...,
  stylesheet: ...,
  resources: {
    ConnectionPointImageUrl: ...
  }
})
```

Then, the users can provide their own images while the default images are also available.

Let me know what you think!

Reply:
Yes, this will be a better way. This can apply to other images referenced within the code.
 I suggest we manage it in a dedicated Discussion/Pull Request as it goes beyond the scope of the fix we want to do here.

Reply:
I think if we wanted to be (reasonably) back-compatible, we could set `Client.imageBasePath` to be `null` by default and use ${Client.imageBasePath}/point.gif if specified by the user or `pointImg` if not (i.e. 

    pointImage = new Image(Client.imageBasePath != null ? `${Client.imageBasePath}/point.gif` : pointImg, 5, 5);

I feel more generally, it would be nice to allow setting specific user subclasses of handlers so that you could have different settings for different graphs on the same page. While subclasses might be a fairly verbose solution, it at least is fairly consistent and haven't been able to come up with a better solution.

Reply:
I think it is best to stay away from global configuration(the previous way) since it is separated from the actual Graph instantiation. When we want to create multiple different graphs on the same page, there has to be separate <script> tags containing `Client.imageBasePath = ...` which also could result in race conditions and so on.

I agree with @tbouffard that this should be an another topic. I'll merge this PR first, and let's discuss on this somewhere else.